src/control/crawler.c

```diff
@@ -214,116 +216,257 @@ static void dt_control_crawler_response_callback(GtkWidget *dialog, gint respons
   free(gui);
 }
 
-// unselect the "select all" toggle
-static void _clear_select_all(dt_control_crawler_gui_t *gui)
+
+static void _delete_selected_rows(GList *rr_list, GtkTreeModel *model)
 {
-  g_signal_handler_block(G_OBJECT(gui->select_all), gui->select_all_handler_id);
-  gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(gui->select_all), FALSE);
-  g_signal_handler_unblock(G_OBJECT(gui->select_all), gui->select_all_handler_id);
+  // Remove TreeView rows from rr_list. It needs to be populated before
+  GList *node;
+  for(node = rr_list; node != NULL; node = g_list_next(node))
```

Comment:
`for(GList *node = ..)`

and remove decl just above.

Reply:
done